.github/workflows/get-started-dotnet.yml

```diff
@@ -34,7 +34,31 @@ jobs:
       uses: actions/setup-dotnet@v3
       with:
         dotnet-version: 7.x
+
+    - name: Restore dependencies
```

Comment:
why not use the variable from the matrix instead?

Reply:
Oh, sorry, that's the reason for this change - it doesn't work :D. See https://github.com/microsoft/kiota-samples/actions/runs/4556131211/jobs/8036094062. It's not setting the CWD correctly when trying to use a matrix.

I just realized that I neglected to remove the matrix., I'll push a change.